### PR TITLE
Fixed the messaging styling

### PR DESCRIPTION
**Pull request by julietrinh1**
- the page is more responsive
- the name of the user chat is displayed 
- the buttons and text area are displayed like in the figma
- the messages display are aligned on the left for the receiver and on the right for the sender

### CODE REVIEW

**Review by fatema-a — CHANGES_REQUESTED** [review]
job postings was fully deleted needs to be added back

**Comment by julietrinh1** [comment]
thank you for letting me know, a mistake from my part. I have reverted the file and taken the file from main to make sure I do not overwrite anything else than the messaging part.

File: frontend/src/components/DirectMessage/Messaging.css
Comment (LeonZhang7) [inline frontend/src/components/DirectMessage/Messaging.css:1]: I see the styling fixes you have done for messaging

File: frontend/src/components/Jobs/JobPostings.js
[the remark this answers is not in the document]
Comment (LeonZhang7) [reply frontend/src/components/Jobs/JobPostings.js:1]: thank you for reverting the changes on JobPosting.js

File: frontend/src/components/DirectMessage/Messaging.js
Comment (LeonZhang7) [inline frontend/src/components/DirectMessage/Messaging.js:1]: I see the adjustments on the messaging.js frontend

**Review by LeonZhang7 — APPROVED** [review]
Thank you for fixing the styling of messaging

**Review by LeonZhang7 — APPROVED** [review]
approved, will merge

**Review by fatema-a — APPROVED** [review]

**Pull request merged**